plugins/templates/README.md

```diff
@@ -27,37 +27,37 @@ server.register({
 
 ### Routes
 
-#### Get all templates
-`page` and `count` optional
+#### Template
+##### Get all templates
 
-`GET /templates?page={pageNumber}&count={countNumber}`
+`GET /templates`
 
-#### Get single template
+##### Get a single template
 
 You can get a single template by providing the template name and the specific version or the tag.
 
 `GET /templates/{name}/{tag}` or `GET /templates/{name}/{version}`
 
-**Arguments**
+###### Arguments
 
 'name', 'tag' or 'version'
 
 * `name` - Name of the template
 * `tag` - Tag of the template (e.g. `stable`, `latest`, etc)
 * `version` - Version of the template
```

Comment:
Maybe we could call out that we'll autobump versions for them when they create templates.